tutorials/fundamentals/concat-num-to-string.md

```diff
@@ -0,0 +1,10 @@
+If you're combining elements [you should not use `concat()`](/tutorials/fundamentals/string-concat).
```

Comment:
Why not use `concat()`? It's perfectly valid to use either or.

Reply:
In the article I'm linking it seemed that it was the least desirable option.